README.md

```diff
@@ -94,15 +94,14 @@ the conf file will be mounted into docker container at runtime to PGSERVICEFILE
 
 To active https with LetsEncrypt just follow the following instructions:
 
-- move `config/_nginx/django_ssl.conf` to `config/nginx/django_ssl.conf`
-- check the domain name in the `.env` file and in `config/nginx/django_ssl.conf`
+- uncomment ssl section within `config/nginx/django.conf`
+- update `WEBGIS_PUBLIC_HOSTNAME` environment variable within the `.env` file
 - run: `docker pull certbot/certbot`
 - launch `./run_certbot.sh`
-- activate 301 redirect into `config/nginx/django.conf`
 - restart compose
 - make sure the certs are renewed by adding a cron job with `crontab -e` and add the following line:
   `0 3 * * * /<path_to_your_docker_files>/run_certbot.sh`
```

Comment:
So why do we need to define a certbot container in `docker-compose.yml` as well?

https://github.com/g3w-suite/g3w-suite-docker/blob/e5695a13af0a0a3a9c8802945027e7bafdc035b6/docker-compose.yml#L95-L100


https://github.com/g3w-suite/g3w-suite-docker/blob/e5695a13af0a0a3a9c8802945027e7bafdc035b6/run_certbot.sh#L24-L31

Reply:
@Raruto Right, I'll do some tests